extensions/hibernate-reactive/deployment/src/test/java/io/quarkus/hibernate/reactive/compatibility/ORMReactiveCompatbilityDifferentNamedDataSourceNamedPersistenceUnitBothUnitTest.java

```diff
@@ -27,10 +27,8 @@ public class ORMReactiveCompatbilityDifferentNamedDataSourceNamedPersistenceUnit
             .setForcedDependencies(List.of(
                     Dependency.of("io.quarkus", "quarkus-jdbc-postgresql-deployment", Version.getVersion()) // this triggers Agroal
             ))
-            .withConfigurationResource("application-unittest-both-named-different.properties")
-
             // Reactive named datasource
-            .overrideConfigKey("quarkus.datasource.\"named-datasource-reactive\".jdbc", "false")
+            .overrideConfigKey("quarkus.datasource.\"named-datasource-reactive\".jdbc", "true")
```

Comment:
`named-datasource-reactive` is jdbc = false, reactive = true

`named-datasource-blocking` is jdbc = true, reactive = false